servicemesh-install/topics/install-overview.adoc

```diff
@@ -0,0 +1,44 @@
+[[servicemesh_install_overview]]
+= {ProductName} Installation Overview
+
+The {ProductName} installation process creates two different projects (namespaces):
+
+* istio-operator project (1 pod)
+* istio-system project (16 pods)
+
+You first create the `istio-operator` project and create a Kubernetes operator.  This operator defines and monitors a custom resource that manages the deployment, updating, and deletion of the Service Mesh components.
+
+The operator creates an Ansible job that runs an Ansible playbook that performs the following installation and configuration tasks automatically:
+ 
+* Creates the `istio-system` namespace
+* Creates the `openshift-ansible-istio-installer-job` which installs the following Istio components:
+** istio-citadel
+** istio-egressgateway
+** istio-galley
+** istio-ingressgateway
+** istio-pilot
+** istio-policy
+** istio-sidecar-injector
+** istio-statsd-prom-bridge
+** istio-telemetry
+* Installs Elasticsearch
+* Installs Grafana
+* Installs the following Jaeger components:
+** jaeger-agent
+** jaeger-collector
+** jaeger-query
+* Installs Prometheus
+* Creates a `devex` project and installs the Fabric8 launcher into that project.
+* Adds the cluster admin role to the admin user (`oc adm policy add-cluster-role-to-user cluster-admin admin`)
```

Comment:
The installer does not grant cluster-admin to the admin role however it will grant this cluster role to the user being used to run the fabric8 launcher if installed

Reply:
@JStickler It's the openshift user which will be used to run the boosters.